scripts/create_config.py

```diff
@@ -1,41 +1,100 @@
 #!/usr/bin/env python
 
 import json
-import os
-from datetime import datetime, date
+from os.path import isfile
+from hashlib import sha512
+from pathlib import Path
+from urllib.request import urlretrieve
 
+from asreviewcontrib.statistics import DataStatistics
 
-def main():
-    config_fp = os.path.join("..", "asreviewcontrib", "covid19", "config.json")
-    covid_meta = {
-        "name": "cord19",
-        "title": "CORD-19",
-        "description": "A Free dataset on publications on the corona virus.",
-        "last_update": str(datetime.now()),
-        "license": "Covid dataset license",
-        "link": "https://pages.semanticscholar.org/coronavirus-research",
+
+def create_config(data_fp, last_update, complete=False, url=None):
```

Comment:
Move this to the core? It sounds like a very useful function for all extension packages.

Reply:
Hmmm, I'm not sure which part is generic enough to be in core?